pkgs/build-support/emacs/wrapper.nix

```diff
@@ -220,6 +225,7 @@ runCommand
         --subst-var-by wrapperSiteLispNative "$deps/share/emacs/native-lisp" \
         --subst-var-by prog "$emacs/Applications/Emacs.app/Contents/MacOS/Emacs"
       chmod +x $out/Applications/Emacs.app/Contents/MacOS/Emacs
+      wrapProgramBinary $out/Applications/Emacs.app/Contents/MacOS/Emacs
```

Comment:
Is this thing also used as a shebang?  The command line `emacs` has already be wrapped above.

Reply:
I did it for completeness and consistency, not sure what people actually do with their scripts.

Honestly, making things user friendly for Emacs users is always a total shot in the dark because they are so varied, and many of them are not programmers at all. Although I have a vague suspicion that share will be far lower in the Nix Users ∩ Emacs Users community 😂

All in all I just thought it'd be nice to be consistent, and I don't really see a big cost to it here, but this isn't exactly a precious PR to me so if you disagree I can be swayed about as easily as a blade of grass.

Reply:
My opinion is that an end user of a software should be able to use it in Nixpkgs the same way she would use it in Ubuntu or Arch - at least ideally, and obviously changing some bits like `apt install -> nix build`.

We can't even suppose Emacs ∩ Nixpkgs is composed of programmers only,  because [SnowflakeOS](https://snowflakeos.org/).

So, I  am not against this here.